CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## Unreleased 
+## Unreleased
 
-docs: Improve code doc on start and endSession (#1098)
+- feat: Expose SentryDebugImageProvider (#1094)
+- docs: Improve code doc on start and endSession (#1098)
```

Comment:
is this part of this PR?

Reply:
This stems from merging master into this branch and resolving conflicts. So we are good to go.